src/storeItems.js

```diff
@@ -0,0 +1,35 @@
+import { todoList, display } from './status.js';
+
+const inp = document.getElementById('inpId');
```

Comment:
Please use a  descriptive variable name because it makes it easier for others (or even yourself) to understand the purpose and use of a variable within the code.